opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -96,12 +96,31 @@ def test_get_handle(self):
             self.assertEqual(metric.handles.get(label_values), handle)
 
 
-class TestCounterHandle(unittest.TestCase):
-    def test_update(self):
-        handle = metrics.CounterHandle(float, True, False)
-        handle.update(2.0)
-        self.assertEqual(handle.data, 2.0)
+class TestCounter(unittest.TestCase):
+    def test_add(self):
+        metric = metrics.Counter("name", "desc", "unit", int, ("key",))
+        handle = metric.get_handle(("value",))
+        metric.add(("value",), 3)
```

Comment:
put a second `add` call to be sure it is actually adding and not just assigning?